src/source/PeerConnection/SessionDescription.c

```diff
@@ -313,7 +313,7 @@ PCHAR fmtpForPayloadType(UINT64 payloadType, PSessionDescription pSessionDescrip
     CHAR payloadStr[MAX_SDP_ATTRIBUTE_VALUE_LENGTH];
 
     MEMSET(payloadStr, 0x00, MAX_SDP_ATTRIBUTE_VALUE_LENGTH);
-    SPRINTF(payloadStr, "%" PRId64, payloadType);
+    SNPRINTF(payloadStr, SIZEOF(payloadStr), "%" PRId64, payloadType);
```

Comment:
Add checks for SNPRINTF return value, for example see usages in `iceCandidateSerialize`.  Most cases if we failed to write the whole thing we return an error / bail out.